microscope/reporter_tokenizer.py

```diff
@@ -46,14 +49,7 @@ def tokenize(text: str) -> List[str]:
 
 
 def _tokenize(text: str) -> List[str]:
-    # add extra space to make things easier
-    text = " " + text + " "
-
     # get rid of all the annoying underscores in text from pdfs
     text = re.sub(r"__+", "", text)
```

Comment:
This function has gotten so small it might as well just go away, right? Maybe move this line under the excess whitespace remover, and then put the `.strip().split()` into the `extend` call? Then we can just remove `_tokenize`?

Reply:
Good idea, fixed.

Reply:
FWIW, I'm not sure that line is necessary either. I intended to remove it in https://github.com/freelawproject/courtlistener/pull/1226 for unrelated reasons and don't remember encountering any negative effects. Just something to think about as we continue to refactor.